src/Network/DogStatsd.hs

```diff
@@ -0,0 +1,53 @@
+module Network.DogStatsd (
+ dogStatsdClient
+, increment
+, decrement
+, count
+, gauge
+, timing
+, histogram
+) where
+
+import Network.Statsd (Stat, Type(..), fmtDatagram)
+import Network.Statsd.UdpClient(UdpClient(..), fromURI, send)
```

Comment:
Yup, you're right. Removed from here and the regular StatsdClient in 41f70682c3cfe6a001ab14d47d7b628ff6c36a2f